cli/src/cli.rs

```diff
@@ -969,13 +989,15 @@ pub fn process_command(config: &CliConfig) -> ProcessResult {
             nonce_authority,
             memo,
             new_authority,
+            compute_unit_price,
         } => process_authorize_nonce_account(
             &rpc_client,
             config,
             nonce_account,
             *nonce_authority,
             memo.as_ref(),
             new_authority,
+            compute_unit_price,
```

Comment:
```suggestion
            compute_unit_price.as_ref(),
```
... if you make the type change I've suggested below.